test.js

```diff
@@ -18,4 +18,14 @@ describe('bang-bang', () => {
       expect(value).to.be('Hello world!')
     })
   })
+
+  describe('with more that one occurence of "!"', () => {
+    it('returns "Hello world!!!"', () =>
+      {
+        const value = bangbang('Hello world', { times: 3 })
+
+        expect(value).to.be('Hello world!!!')
```

Comment:
Ne pourrait-on pas "optimiser" cette fonction de vérification et l'écrire une fois pour chaque cas (0,1,2 ou 3 "!") au lieu d'écrire ces lignes 4 fois pour les 4 cas différents.

Histoire qu'à la place de ces 4 ensemble de lignes il n'y ait que:
expect(value).to.be('Hello world'times*'!')

Reply:
Comme pour le commentaire au-dessus de @JojoDz, idéalement, si on avait travaillé ensemble sur le projet, je serais sortie de l'explication écrite pour qu'on ait une discussion ensemble. Je tente une explication tout de même :)

Contrairement à la PR #5 où on peut ajouter aléatoirement entre 0 et 3 points d'exclamations, cette PR permet d'ajouter autant de `!` que l'on veut à la fin de notre chaine de caractères. J'ai l'impression qu'une partie de ton commentaire se refère plutôt à la PR #5. Si c'est le cas, tu pourras voir que j'ai changé le test dans cette PR pour correspondre à la fonctionnalité donnée.

Pour cette PR-ci, je trouve ton commentaire très intéressant à propos de la structure de mon test.

Ce que je tente de vérifier, c'est qu'en passant le paramètre `{ times: 3 }` à la fonction, trois points d'exclamation sont bien ajoutés à la chaine de caractère. Effectivement, je ne teste pas d'autres nombres. Ce qui rend mon test peu robuste. Les options possibles pour moi ici pour améliorer mon test : 
- tester une série de nombre pour l'option `times`. Dans ce cas, effectivement, ta proposition de factorisation est très utile,
- tester le cas où on n'a mis autre chose qu'un `Number` pour l'option `times` (par exemple, `String`, `null`, `undefined`)

Penses-tu que je devrais ajouter ces autres tests ?